components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -164,27 +172,27 @@ - (void)layoutLandscapeModeWithBottomNavSize:(CGSize)bottomNavSize
                               containerWidth:(CGFloat)containerWidth {
   switch (self.alignment) {
     case MDCBottomNavigationBarAlignmentJustified:
-      [self sizeContainerViewItemsDistributed:YES
+      [self sizeitemsLayoutViewItemsDistributed:YES
                             withBottomNavSize:bottomNavSize
                                containerWidth:containerWidth];
       self.titleBelowItem = YES;
       break;
     case MDCBottomNavigationBarAlignmentJustifiedAdjacentTitles:
-      [self sizeContainerViewItemsDistributed:YES
+      [self sizeitemsLayoutViewItemsDistributed:YES
                             withBottomNavSize:bottomNavSize
                                containerWidth:containerWidth];
       self.titleBelowItem = NO;
       break;
     case MDCBottomNavigationBarAlignmentCentered:
-      [self sizeContainerViewItemsDistributed:NO
+      [self sizeitemsLayoutViewItemsDistributed:NO
                             withBottomNavSize:bottomNavSize
                                containerWidth:containerWidth];
       self.titleBelowItem = YES;
       break;
   }
 }
 
-- (void)sizeContainerViewItemsDistributed:(BOOL)itemsDistributed
+- (void)sizeitemsLayoutViewItemsDistributed:(BOOL)itemsDistributed
```

Comment:
_Tip:_ Shouldn't it be `sizeItemsLayoutViewItemsDistributed..`? `sizeItems` ~`sizeitems`~?

Reply:
Good catch - that was a find/replace mistake.